docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/grafana-alloy/page.mdx

---
title: 'Grafana Alloy'
description: 'Forward telemetry from Grafana Alloy into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Grafana Alloy

Grafana Alloy is a telemetry collector built on OpenTelemetry that gathers and forwards logs, metrics, and traces. It integrates tightly with the Grafana ecosystem for observability pipelines.

## Using Grafana Alloy with GlassFlow

GlassFlow can ingest telemetry forwarded by Grafana Alloy into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Grafana Alloy connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Grafana Alloy connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
